examples/managed_instance_group/main.tf

```diff
@@ -91,9 +132,10 @@ resource "google_compute_firewall" "lb-to-instances" {
 
   allow {
     protocol = "tcp"
-    ports    = [var.image_port]
+    ports    = [
+      var.image_port]
```

Comment:
See above RE: moving trailing `]` to separate line

Reply:
@kopachevsky @Jberlinsky 
[Fixed](https://github.com/terraform-google-modules/terraform-google-container-vm/compare/21506b0e8d88a8bee72d02dc63094b8276ccd288..b6d0f962d7a2074d4392d31a3368b8c6808df77b)